### PR TITLE
l10n_ch: fixed translations

### PR DESCRIPTION
**Pull request by rgo-odoo**
Task: <a href="https://www.odoo.com/web#action=4043&cids=2&id=2391988&menu_id=4720&model=project.task&view_type=form">2391988</a>
Related: odoo/enterprise#15079
Related(master): odoo/odoo#62152

### CODE REVIEW

File: addons/l10n_ch/i18n_extra/de.po [hunk 1]
```diff
@@ -70,78 +73,88 @@ msgstr "2.5% Inkl."
...
 #: model:account.tax.report.line,name:l10n_ch.account_tax_report_line_chtax_299
 msgid "299 Chiffre d'affaires imposable (ch.200 moins ch.289)"
-msgstr ""
+msgstr "299 Steuerbarer Gesamtumsatz (Ziff. 200 abzüglich Ziff. 289)"
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_37
 msgid "3.7%"
-msgstr "3.7%"
+msgstr "0.037"
```
Comment (Yenthe666) [inline addons/l10n_ch/i18n_extra/de.po:157]: You sure? 😅

**Comment by fclementic2c** [comment]
I just want to thank guys to work on this :) Many thanks 👍🏻

File: addons/l10n_ch/i18n_extra/de.po [hunk 2]
```diff
@@ -40,27 +38,32 @@ msgstr "0% Exkl."
...
+#. module: l10n_ch
+#: model:account.tax.template,description:l10n_ch.vat_100_import_invest
+msgid "100% imp.invest."
+msgstr ""
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_25
 msgid "2.5%"
-msgstr "2.5%"
+msgstr "0.025"
```
Comment (jco-odoo) [inline addons/l10n_ch/i18n_extra/de.po:66]: @rgo-odoo This is still weird

Comment (rgo-odoo) [reply addons/l10n_ch/i18n_extra/de.po:66]: @jco-odoo actually it was proposed to be changed like that, see column `7M` of `TAX` sheet from task pad.

Comment (jco-odoo) [reply addons/l10n_ch/i18n_extra/de.po:66]: @rgo-odoo That is an error.  (probably to do with the spreadsheet application)

Comment (jco-odoo) [reply addons/l10n_ch/i18n_extra/de.po:66]: 2.5 %  is definitely easier to read.

**Review by yvaucher — APPROVED** [review]

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**